packages/sanity/src/core/scheduledPublishing/components/scheduleItem/PreviewWrapper.tsx

```diff
@@ -154,7 +154,7 @@ const PreviewWrapper = (props: Props) => {
                     <DocumentStatusIndicator
                       draft={previewState?.draft}
                       published={previewState?.published}
-                      versions={previewState?.versions}
```

Comment:
if schedulePublishing should not support versions I wonder why we did this 🤔 
I just want to double check that we are sure about this change?